org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/parser/AnnotationDietRecoveryTest.java

```diff
@@ -40,13 +39,13 @@
 public class AnnotationDietRecoveryTest extends AbstractCompilerTest {
 	private static final boolean CHECK_ALL_PARSE = true;
 	public static boolean optimizeStringLiterals = false;
-	public static long sourceLevel = ClassFileConstants.JDK1_3; //$NON-NLS-1$
+	public static long sourceLevel = CompilerOptions.getFirstSupportedJdkLevel(); //$NON-NLS-1$
```

Comment:
Field can be gotten rid of ? Same named field is deleted by this PR in a few other *DietRecovery tests.

Reply:
If it is left, it is still in use. I've only removed unused stuff

Reply:
> If it is left, it is still in use. I've only removed unused stuff

I checked before asking :) There are no references to `org.eclipse.jdt.core.tests.compiler.parser.AnnotationDietRecoveryTest.sourceLevel`

Reply:
OK, so I will remove it. I assume it is public static to it wasn't noticed by me because no warning was reported in that case.